gapis/api/templates/vulkan_gfx_api_extras.tmpl

```diff
@@ -55,17 +55,7 @@
 ¶
   const char kVirtualSwapchainLayerName[] = "VirtualSwapchain";
   const char kGraphicsSpyLayerName[] = "GraphicsSpy";
-  const char* kValidationLayerNames[] = {
-    // Meta layers
-    "VK_LAYER_KHRONOS_validation",
-    "VK_LAYER_LUNARG_standard_validation",
-    // Regular layers
-    "VK_LAYER_GOOGLE_threading",
-    "VK_LAYER_LUNARG_parameter_validation",
-    "VK_LAYER_LUNARG_object_tracker",
-    "VK_LAYER_LUNARG_core_validation",
-    "VK_LAYER_GOOGLE_unique_objects",
-  };
+  const char* kValidationLayerNames[] = { "VK_LAYER_KHRONOS_validation" };
```

Comment:
This will breaking replay of application that ends up requesting the old validation layers.